packages/checkout-com/src/useCko.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable camelcase, @typescript-eslint/camelcase */
+
+import { createContext } from './payment';
+import { Configuration } from './configuration';
+import { ref } from '@vue/composition-api';
+import useCkoCard from './useCkoCard';
+
+const error = ref(null);
+const availableMethods = ref([]);
+
+interface PaymentMethods {
+  card?: boolean;
+  klarna?: boolean;
+  paypal?: boolean;
+}
+
+interface PaymentMethodsConfig {
+  card?: Omit<Configuration, 'publicKey'>;
+  klarna?: any;
+  paypal?: any;
+}
+
+enum PaymentMethod {
+  NOT_SELECTED = 0,
+  CARD,
+  SAVED_CARD,
+  KLARNA,
+  PAYPAL
+}
+
+const selectedPaymentMethod = ref(PaymentMethod.NOT_SELECTED);
+
+const useCko = () => {
+  const loadAvailableMethods = async (reference) => {
+    try {
+      const response = await createContext({ reference });
+      availableMethods.value = [
+        ...response.data.apms,
+        { name: 'card' }
```

Comment:
And here yet another format. So finally, `availableMethods` is:
```ts
{ name: string, [prop]:string }[]
```
Correct?

Reply:
It is:
```
{ name: string, [key: string]: any }
```